### PR TITLE
fix(bpmn): unregister `commandStack.changed` listener

### LINKED ISSUE #2732: Modeler loses the active tab indicator
__Describe the Bug__

With recent Nightlies of the Modeler and also with the Modeler 5.0.0-alpha version, the active tab indicator for the opened editor (the dark grey line) gets lost from time to time.

__Steps to Reproduce__

Did not yet find out how to reproduce.

Here is a screenshot:

Screen Shot 2022-02-08 at 15 56 14

__Expected Behavior__

The active tab indicator should remain.

__Environment__

 * Version: 5.0.0-nightly.20220202, Camunda Modeler Version: 5.0.0-alpha.1
 * Operating System: Mac OS 12.1.0 
 * Plugins: Cloud Connect
 * Execution Platform: BPMN - Camunda Platform

### PR DESCRIPTION
**Pull request by marstamm**
closes #2732

### CODE REVIEW

File: client/src/app/tabs/bpmn/BpmnEditor.js [hunk 1]
```diff
@@ -214,7 +214,11 @@ export class BpmnEditor extends CachedComponent {
 
     modeler[fn]('minimap.toggle', this.handleMinimapToggle);
 
-    modeler[ fn ]('commandStack.changed', LOW_PRIORITY, this.handleLintingDebounced);
+    if (fn === 'on') {
```
Comment (nikku) [inline client/src/app/tabs/bpmn/BpmnEditor.js:217]: We'd need to fix this for all our editors, right?

Comment (marstamm) [reply client/src/app/tabs/bpmn/BpmnEditor.js:217]: I fixed it for cloud and platform editors, form editor already took care of it.

These are the only ones that use linting as far as I could tell

Comment (nikku) [reply client/src/app/tabs/bpmn/BpmnEditor.js:217]: No other editors register / de-register with priority?

Comment (marstamm) [reply client/src/app/tabs/bpmn/BpmnEditor.js:217]: Found only one instance in the cmmn editor, which I fixed in [1613166](https://github.com/camunda/camunda-modeler/pull/2755/commits/1613166470397d0f3ca8e4755217ca420f9ec2e4)

**Comment by nikku** [comment]
Gold or metal medal? Makes a difference.

**Comment by marstamm** [comment]
Obviously Metal :guitar:

**Pull request merged**